### PR TITLE
Bugfix/dm 2674 failed changing external microservice service key y2024

### PR DESCRIPTION
**Pull request by shaikhahme**

### CODE REVIEW

File: content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-2674
+version: 10.18.540.74
+---
+
+Changing the external microservice was not working as expected because the LWM2M agent would still point the custom
+actions to the old microservice as the service-key was not being updated.This LWM2M agent version fixes the bug by
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md:19]:
```suggestion
actions to the old microservice as the service key was not being updated. Now
```

File: content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+updating the service-key when the external microservice is changed.
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md:20]:
```suggestion
the service key is updated properly when the external microservice is changed.
```

File: content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  - value: component-1KLUzmqfe
+    label: LWM2M
+build_artifact:
+  - value: tc-QHwMfWtBk7
+    label: cumulocity
+ticket: DM-2674
+version: 10.18.540.74
+---
+
+In LWM2M device protocols resource's custom action configuration, the exchange to another external decoder microservice
```
Comment (BeateRixen) [inline content/change-logs/device-management/cumulocity-10-20-316-0-Fixed-issue-with-changing-microservice-service-key.md:18]:
```suggestion
In the custom action configuration of LWM2M device protocol resources, the exchange to another external decoder microservice
```
"LWM2M device protocols resource's custom action configuration" is a very long term, hard to understand :-)

**Review by BeateRixen — APPROVED** [review]

**Review by Oezge-Akin — COMMENTED** [review]
Could you correct the version in the filename?

**Review by Oezge-Akin — APPROVED** [review]

File: content/change-logs/device-management/cumulocity-10-18-540-74-Fixed-issue-with-changing-microservice-service-key.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+date:
```
Comment (Oezge-Akin) [inline content/change-logs/device-management/cumulocity-10-18-540-74-Fixed-issue-with-changing-microservice-service-key.md:2]:
```suggestion
date: 2024-03-18
```

**Comment by BeateRixen** [comment]
@shaikhahme Can we merge this?

**Pull request merged**